addons/website_slides/controllers/main.py

```diff
@@ -797,6 +797,26 @@ def create_slide(self, *args, **post):
             else:
                 values['category_id'] = post['category_id'][0]
 
+        new_certification = False
+        if post.get('survey_id'):
```

Comment:
Just to be sure, we could try / except the survey creation and return an error for the JS to display if people mistake it somehow.